cmd/runtimecfg/node-ip.go

```diff
@@ -83,6 +84,13 @@ func show(cmd *cobra.Command, args []string) error {
 }
 
 func set(cmd *cobra.Command, args []string) error {
+	// If primary ip address was already created, it means that nodeip-configuration has run already and no need to
+	// choose new ip, we should leave same configuration as we already set
+	if ip, err := config.GetIpFromFile(nodeIpFile); err == nil {
```

Comment:
nit: What happens if we get err here, just ignore?

Reply:
Yep. Failed to read file go to the main logic

Reply:
Adding `err != nil` and `return err` early seems okay too.

Reply:
I don't actually think this will be right in that case. There is no point of stopping run because if this error.
We should continue running till the end as this can actually fix this error by , lets say, writing ip to the file that somehow was written wrong previously or touched by user for unknown reason.
I really thinks that code in such critical service should be resilient to errors as much as possible and we should continue running till we can, we will not win anything in falling 0.5 seconds before.

Reply:
Sure, that's okay for me.